assets/javascripts/modules/checkout/validations.js

```diff
@@ -46,12 +47,12 @@ define(['$', 'modules/checkout/form-elements', 'modules/checkout/regex'], functi
 
 
     var validatePaymentDetails = function () {
-        var accountNumberValid = form.$ACCOUNT.val() != ''
+        var accountNumberValid = form.$ACCOUNT.val() !== ''
             && form.$ACCOUNT.val().length <= 10
-            && isNumber(form.$ACCOUNT.val());
+            && regex.isNumber(form.$ACCOUNT.val());
         toggleError(form.$ACCOUNT_CONTAINER, !accountNumberValid);
 
-        var holderNameValid = form.$HOLDER.val() != ''
+        var holderNameValid = form.$HOLDER.val() !== ''
```

Comment:
@tudorraul You might want to give this a look to check if you do want strict equality. You nearly always do, but you might have reason to not do that.